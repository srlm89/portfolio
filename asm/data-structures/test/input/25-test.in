ERASE actual/25-test.out
INSERT 24 0.602742 The X Files - I Want To Believe    720p
PRINT_IN_ORDER actual/25-test.out
INSERT 7 9.538014 Lord Of The Ring  - Return Of The King   720p
INSERT 21 3.424967 Drag Me To Hell [720p]
REMOVE_ID 24
PRINT_IN_ORDER actual/25-test.out
INSERT 25 9.921503 Jeniffer's Body ( 2009)    1080p
INSERT 12 0.710526 SAW I   720p
INSERT 8 2.189830 Deep Sea
REMOVE_ID 25
PRINT_IN_ORDER actual/25-test.out
INSERT 6 5.428519 1408.2007.720p.BluRay.x264-SiNNERS
INSERT 29 5.033756 Stardust   720p
INSERT 23 3.217133 Ice Age 1 [2002] [720p]
INSERT 30 5.053188 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE
REMOVE_ID 8
REMOVE_ID 6
PRINT_IN_ORDER actual/25-test.out
INSERT 26 2.685497 WALL-E.720p.BluRay.x264-iNFAMOUS
INSERT 10 7.039659 Yes Man   720p
INSERT 22 0.005440 Danny the Dog   1080p
INSERT 20 6.022834 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL
INSERT 38 3.998877 Land Of the Lost   720p
REMOVE_ID 38
REMOVE_ID 10
PRINT_IN_ORDER actual/25-test.out
INSERT 35 5.307148 Bad Boy I [1995] [720p]
INSERT 0 8.684771 Doomsday 720p
INSERT 18 0.076079 The Orphan [720]
INSERT 39 4.536402 Pursuit_Of_Happyness[2006]   720p
INSERT 19 0.835365 The Orphan [720]
INSERT 11 8.208236 Starship Troopers 2  720p
REMOVE_ID 29
REMOVE_ID 0
REMOVE_ID 35
PRINT_IN_ORDER actual/25-test.out
INSERT 4 3.399181 X-Men 3   1080p
INSERT 1 7.845460 911 Mysteries Demolition    720p
INSERT 3 2.388278 The Final Destination 4 [2009] [720p]
INSERT 48 1.274903 PBS
INSERT 2 0.968407 Angels & Demons [2009] [1080p]
INSERT 47 2.736238 Ice Age 1 [2002] [720p]
INSERT 36 8.232703 Madagascar 2 Escape 2 Africa [2008] [720p]
REMOVE_ID 7
REMOVE_ID 48
REMOVE_ID 3
PRINT_IN_ORDER actual/25-test.out
INSERT 9 0.366572 XxX.2002.BDRiP.720p.AC3.5.1.READnfo.x264-DEFiNiTiON
INSERT 33 5.245987 Banlieue 13 - Ultimatum [720p]
INSERT 50 8.326659 Incredibles, The 1080p
INSERT 5 3.606615 Kill Shot   720p
INSERT 34 4.538381 10,000 BC 720p
INSERT 14 9.657084 SAW V
INSERT 44 8.210330 Shark Tale [2004] [1080p]
INSERT 37 3.865847 A Knight's Tale [2001] [720p]
REMOVE_ID 34
REMOVE_ID 22
REMOVE_ID 23
REMOVE_ID 30
PRINT_IN_ORDER actual/25-test.out
INSERT 31 4.897693 Incredibles, The 1080p
INSERT 13 7.933499 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL
INSERT 45 7.457275 Kung Fu Panda [2008]   720p
INSERT 28 2.841911 Invisible.Target.720p.Bluray.x264-SEPTiC
INSERT 52 9.351746 Batman The Dark Knight   720p
INSERT 62 8.286349 Death Race   720p
INSERT 59 0.322636 Seven Pounds    720p
INSERT 64 2.791038 Elite Squad 1080p
INSERT 40 2.582334 contents.txt
REMOVE_ID 11
REMOVE_ID 45
REMOVE_ID 4
REMOVE_ID 40
PRINT_IN_ORDER actual/25-test.out
