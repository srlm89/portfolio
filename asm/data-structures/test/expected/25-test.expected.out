[ 24 0.602742 The X Files - I Want To Believe    720p ]
[ 7 9.538014 Lord Of The Ring  - Return Of The King   720p ][ 21 3.424967 Drag Me To Hell [720p] ]
[ 7 9.538014 Lord Of The Ring  - Return Of The King   720p ][ 8 2.189830 Deep Sea ][ 12 0.710526 SAW I   720p ][ 21 3.424967 Drag Me To Hell [720p] ]
[ 7 9.538014 Lord Of The Ring  - Return Of The King   720p ][ 12 0.710526 SAW I   720p ][ 21 3.424967 Drag Me To Hell [720p] ][ 23 3.217133 Ice Age 1 [2002] [720p] ][ 29 5.033756 Stardust   720p ][ 30 5.053188 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE ]
[ 7 9.538014 Lord Of The Ring  - Return Of The King   720p ][ 12 0.710526 SAW I   720p ][ 20 6.022834 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL ][ 21 3.424967 Drag Me To Hell [720p] ][ 22 0.005440 Danny the Dog   1080p ][ 23 3.217133 Ice Age 1 [2002] [720p] ][ 26 2.685497 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 29 5.033756 Stardust   720p ][ 30 5.053188 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE ]
[ 7 9.538014 Lord Of The Ring  - Return Of The King   720p ][ 11 8.208236 Starship Troopers 2  720p ][ 12 0.710526 SAW I   720p ][ 18 0.076079 The Orphan [720] ][ 19 0.835365 The Orphan [720] ][ 20 6.022834 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL ][ 21 3.424967 Drag Me To Hell [720p] ][ 22 0.005440 Danny the Dog   1080p ][ 23 3.217133 Ice Age 1 [2002] [720p] ][ 26 2.685497 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 30 5.053188 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE ][ 39 4.536402 Pursuit_Of_Happyness[2006]   720p ]
[ 1 7.845460 911 Mysteries Demolition    720p ][ 2 0.968407 Angels & Demons [2009] [1080p] ][ 4 3.399181 X-Men 3   1080p ][ 11 8.208236 Starship Troopers 2  720p ][ 12 0.710526 SAW I   720p ][ 18 0.076079 The Orphan [720] ][ 19 0.835365 The Orphan [720] ][ 20 6.022834 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL ][ 21 3.424967 Drag Me To Hell [720p] ][ 22 0.005440 Danny the Dog   1080p ][ 23 3.217133 Ice Age 1 [2002] [720p] ][ 26 2.685497 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 30 5.053188 The.Shawshank.Redemption.1994.1080p.BluRay.x264-CiNEFiLE ][ 36 8.232703 Madagascar 2 Escape 2 Africa [2008] [720p] ][ 39 4.536402 Pursuit_Of_Happyness[2006]   720p ][ 47 2.736238 Ice Age 1 [2002] [720p] ]
[ 1 7.845460 911 Mysteries Demolition    720p ][ 2 0.968407 Angels & Demons [2009] [1080p] ][ 4 3.399181 X-Men 3   1080p ][ 5 3.606615 Kill Shot   720p ][ 9 0.366572 XxX.2002.BDRiP.720p.AC3.5.1.READnfo.x264-DEFiNiTiON ][ 11 8.208236 Starship Troopers 2  720p ][ 12 0.710526 SAW I   720p ][ 14 9.657084 SAW V ][ 18 0.076079 The Orphan [720] ][ 19 0.835365 The Orphan [720] ][ 20 6.022834 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL ][ 21 3.424967 Drag Me To Hell [720p] ][ 26 2.685497 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 33 5.245987 Banlieue 13 - Ultimatum [720p] ][ 36 8.232703 Madagascar 2 Escape 2 Africa [2008] [720p] ][ 37 3.865847 A Knight's Tale [2001] [720p] ][ 39 4.536402 Pursuit_Of_Happyness[2006]   720p ][ 44 8.210330 Shark Tale [2004] [1080p] ][ 47 2.736238 Ice Age 1 [2002] [720p] ][ 50 8.326659 Incredibles, The 1080p ]
[ 1 7.845460 911 Mysteries Demolition    720p ][ 2 0.968407 Angels & Demons [2009] [1080p] ][ 5 3.606615 Kill Shot   720p ][ 9 0.366572 XxX.2002.BDRiP.720p.AC3.5.1.READnfo.x264-DEFiNiTiON ][ 12 0.710526 SAW I   720p ][ 13 7.933499 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL ][ 14 9.657084 SAW V ][ 18 0.076079 The Orphan [720] ][ 19 0.835365 The Orphan [720] ][ 20 6.022834 Bugs.A.Rainforest.Adventure.3D.HD.DVD.REMUX.1080p.VC-1-SEBUL ][ 21 3.424967 Drag Me To Hell [720p] ][ 26 2.685497 WALL-E.720p.BluRay.x264-iNFAMOUS ][ 28 2.841911 Invisible.Target.720p.Bluray.x264-SEPTiC ][ 31 4.897693 Incredibles, The 1080p ][ 33 5.245987 Banlieue 13 - Ultimatum [720p] ][ 36 8.232703 Madagascar 2 Escape 2 Africa [2008] [720p] ][ 37 3.865847 A Knight's Tale [2001] [720p] ][ 39 4.536402 Pursuit_Of_Happyness[2006]   720p ][ 44 8.210330 Shark Tale [2004] [1080p] ][ 47 2.736238 Ice Age 1 [2002] [720p] ][ 50 8.326659 Incredibles, The 1080p ][ 52 9.351746 Batman The Dark Knight   720p ][ 59 0.322636 Seven Pounds    720p ][ 62 8.286349 Death Race   720p ][ 64 2.791038 Elite Squad 1080p ]
